### ISSUE TITLE
[JENKINS-75333] Contrast ratio requirements 4.5:1 is not met for some texts 

### ISSUE THREAD

**Issue by lemeurherve** [lemeurherve-org/demo#794]
<p>The following text does not meet the contrast ratio of 4.5:1 with its background in latest LTS jenkins 2.479.3:</p>
<ul>
	<li>"This build requires parameters" present in pipelines.</li>
	<li> "No Builds" In the Builds component.</li>
	<li>"Search (Ctrl+K)" in the search box present in page header.</li>
</ul>

<p>Kindly correct this for better accessibility.</p>

<p> </p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=geetika_agarwal">geetika_agarwal</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-75333" target="_blank">Contrast ratio requirements 4.5:1 is not met for some texts </a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Major
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>The following text does not meet the contrast ratio of 4.5:1 with its background in latest LTS jenkins 2.479.3:</p>
<ul>
	<li>"This build requires parameters" present in pipelines.</li>
	<li> "No Builds" In the Builds component.</li>
	<li>"Search (Ctrl+K)" in the search box present in page header.</li>
</ul>

<p>Kindly correct this for better accessibility.</p>

<p> </p></pre>
</details>
</details>

**Comment by lemeurherve** [comment]
<details><summary><i>pushkar_07's <a href="https://issues.jenkins.io/browse/JENKINS-75333?focusedId=452823&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-452823">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>Looking into the issue..</p>
</pre>
</details>

</details>
<p>Looking into the issue..</p>

**Comment by lemeurherve** [comment]
<details><summary><i>pushkar_07's <a href="https://issues.jenkins.io/browse/JENKINS-75333?focusedId=452913&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-452913">comment</a>:</i></summary>


<details><summary><sub><i>Raw content of original comment:</i></sub></summary>

<pre>
<p>Will require some input on what to replace the bg colour as, current bg is transparent and is causing visibility issues</p>
</pre>
</details>

</details>
<p>Will require some input on what to replace the bg colour as, current bg is transparent and is causing visibility issues</p>
